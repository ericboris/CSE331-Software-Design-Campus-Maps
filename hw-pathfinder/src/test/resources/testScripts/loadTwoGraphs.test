# Find the shortest paths in two graphs

CreateGraph g1

AddNode g1 A1
AddNode g1 B1

CreateGraph g2

AddNode g2 A2
AddNode g2 B2

AddEdge g1 A1 B1 1.0
AddEdge g2 A2 B2 2.0

FindPath g1 A1 B1

FindPath g2 A2 B2