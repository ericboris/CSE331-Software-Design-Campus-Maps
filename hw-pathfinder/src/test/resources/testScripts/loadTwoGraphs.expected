# Find the shortest paths in two graphs

created graph g1

added node A1 to g1
added node B1 to g1

created graph g2

added node A2 to g2
added node B2 to g2

added edge 1.000 from A1 to B1 in g1
added edge 2.000 from A2 to B2 in g2

path from A1 to B1:
A1 to B1 with weight 1.000
total cost: 1.000

path from A2 to B2:
A2 to B2 with weight 2.000
total cost: 2.000